### PR TITLE
Adding solution files to all of our device bindings

### PR DESCRIPTION
**Pull request by joperezr**
cc: @pgrawehr @krwq 

This is just adding solution files to all of our device bindings. For future reference, these changes were made through a simple console app and I'll paste next the code in case we ever need to do this again or change it somehow:

```c#
class Program
{
    static void Main(string[] args)
    {
        var deviceDirectories = Directory.GetDirectories(@"F:\git\iot\src\devices\", "*", SearchOption.TopDirectoryOnly);
        Parallel.ForEach(deviceDirectories, (deviceDirectory) =>
        {
            var mainProject = Directory.GetFiles(deviceDirectory, "*.csproj", SearchOption.TopDirectoryOnly);
            if (mainProject.Length != 1)
            {
                Console.WriteLine($"Error: found {mainProject.Length} main projects for device path {deviceDirectory}");
                return;
            }
            RunCommand("new sln", deviceDirectory);

            // Add tests projects to solution
            if (Directory.Exists(Path.Combine(deviceDirectory, "tests")))
            {
                var sampleProjects = Directory.GetFiles(Path.Combine(deviceDirectory, "tests"), "*csproj", SearchOption.AllDirectories);
                foreach (var sampleProject in sampleProjects)
                {
                    RunCommand($"sln add -s tests {sampleProject}", deviceDirectory);
                }
            }

            // Add samples projects to solution
            if (Directory.Exists(Path.Combine(deviceDirectory, "samples")))
            {
                var sampleProjects = Directory.GetFiles(Path.Combine(deviceDirectory, "samples"), "*csproj", SearchOption.AllDirectories);
                foreach (var sampleProject in sampleProjects)
                {
                    RunCommand($"sln add -s samples {sampleProject}", deviceDirectory);
                }
            }

            // Add main project to solution.
            RunCommand($"sln add --in-root {mainProject[0]}", deviceDirectory);
        });
    }

    static void RunCommand(string command, string path)
    {
        ProcessStartInfo processStartInfo = new ProcessStartInfo
        { 
            FileName = @"C:\Program Files\dotnet\dotnet.exe",
            Arguments = command,
            WorkingDirectory = path
        };

        var process = Process.Start(processStartInfo);

        process.WaitForExit();
    }
}
```

### CODE REVIEW

**Comment by joperezr** [comment]
Sounds good, will merge once CI is green.

File: src/devices/ServoMotor/ServoMotor.sln [hunk 1]
```diff
@@ -0,0 +1,70 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0
+MinimumVisualStudioVersion = 15.0.26124.0
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{A6C9C63E-8347-4956-8816-C5D463194BCA}"
```
Comment (pgrawehr) [inline src/devices/ServoMotor/ServoMotor.sln:6]: This is also weird

File: src/devices/Ads1115/Ads1115.sln [hunk 2]
```diff
@@ -0,0 +1,53 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0
+MinimumVisualStudioVersion = 15.0.26124.0
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{9A19ECCF-7171-442B-9C7C-EC16B9A6F96C}"
```
[the remark this answers is not in the document]
Comment (joperezr) [reply src/devices/Ads1115/Ads1115.sln:6]: I'm not sure what do you mean. Would you prefer the solution to ONLY contain the implementation of the binding? In my mind, the idea of having a solution file would be to have all projects you would require while developing that binding available and loaded, and that does include having the samples project (as most likely that is going to be a testing ground for your binding's code) as well as the tests.

File: src/devices/ServoMotor/ServoMotor.sln (hunk 1 above)
Comment (joperezr) [reply src/devices/ServoMotor/ServoMotor.sln:6]: See my comment above, but in general it is very common for developers who are coding a library to also have their unit test project part of the same solution, that is what the intention is here. Most developers that add a unit test project to their binding, will want to have it loaded in VS in order to be able to quickly run and debug the tests right from there.

File: src/devices/Ads1115/Ads1115.sln (hunk 2 above)
[the remark this answers is not in the document]
Comment (joperezr) [reply src/devices/Ads1115/Ads1115.sln:6]: Chatted about this offline with @pgrawehr, there was a confussion on what the samples meant, but that is clear now.

**Pull request merged**